install/kubernetes/Makefile.values

```diff
@@ -38,7 +38,7 @@ export CILIUM_ETCD_OPERATOR_REPO:=quay.io/cilium/cilium-etcd-operator
 export CILIUM_ETCD_OPERATOR_VERSION:=v2.0.7
 export CILIUM_ETCD_OPERATOR_DIGEST:=sha256:04b8327f7f992693c2cb483b999041ed8f92efc8e14f2a5f3ab95574a65ea2dc
 export CILIUM_NODEINIT_REPO:=quay.io/cilium/startup-script
-export CILIUM_NODEINIT_VERSION:=d69851597ea019af980891a4628fb36b7880ec26
+export CILIUM_NODEINIT_VERSION:=5e928f628f9fc644a1d2651d6cd6aecbde0e7acd
```

Comment:
This image is actually published 1 year ago, there is no harm to check in image-tools if there is any upgrade we need to do :thinking:

Reply:
it is so tempting to click the merge button when it's green, but i stayed patient and opened https://github.com/cilium/image-tools/pull/218.

Reply:
yup, it will be great if we can do it in shot, you know, I am backporter this week :laughing:

Reply:
alright let me convert this to draft, let's get https://github.com/cilium/image-tools/pull/218 merged first.